sdk-lib/src/main/java/cash/z/ecc/android/sdk/model/AccountCreateSetup.kt

```diff
@@ -0,0 +1,35 @@
+package cash.z.ecc.android.sdk.model
+
+/**
+ * Wrapper for the create account API based on seed phrase
+ *
+ * @param accountName A human-readable name for the account
+ * @param keySource A string identifier or other metadata describing the source of the seed
+ * @param seed the wallet's seed phrase. This is required the first time a new wallet is set up. For
+ * subsequent calls, seed is only needed if [InitializerException.SeedRequired] is thrown.
+ */
+data class AccountCreateSetup(
+    val accountName: String,
+    val keySource: String?,
+    val seed: ByteArray,
+) {
+    override fun equals(other: Any?): Boolean {
```

Comment:
equals and hashCode are generated by data class here too so we probably do not need to override it

Reply:
The `equals` and `hashCode` are required due to the `ByteArray` parameter in this class.